src/components/app/BottomBox.css

```diff
@@ -42,3 +42,47 @@
   background: url(../../../res/img/svg/close-dark.svg) no-repeat center / 16px
     16px;
 }
+
+.sourceStatusOverlay {
```

Comment:
So, here's what I think would make sense:

```css

.sourceStatusOverlay {
  /**
   * Put the overlay on top of everything in .bottom-main, but centered
   * horizontally and vertically. We center using margin: auto, and enforce
   * maximum sizes so that the line numbers and timings are still visible.
   */

  /* Overlay + centering */
  position: absolute;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  margin: auto;

  /* Size constraints */
  width: max-content;
  min-width: 200px;
  max-width: min(500px, 100%);
  height: min-content;

  /* Content layout */
  display: flex;
  flex-flow: row;
  align-items: center;
  padding: 15px;
  gap: 15px;
  word-break: break-word;

  /* Appearance */
  background-color: rgb(240 240 240 / 0.8);
  border-radius: 10px;
}
```

Unfortunately, our linter enforces a property order which is incompatible with this. After `yarn lint-css --fix` I end up with this, which means that the comments no longer make sense:

```css
.sourceStatusOverlay {
  /**
   * Put the overlay on top of everything in .bottom-main, but centered
   * horizontally and vertically. We center using margin: auto, and enforce
   * maximum sizes so that the line numbers and timings are still visible.
   */

  /* Overlay + centering */
  position: absolute;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;

  /* Content layout */
  display: flex;

  /* Size constraints */
  width: max-content;
  min-width: 200px;
  max-width: min(500px, 100%);
  height: min-content;
  flex-flow: row;
  align-items: center;
  padding: 15px;
  margin: auto;

  /* Appearance */
  background-color: rgb(240 240 240 / 0.8);
  border-radius: 10px;
  gap: 15px;
  word-break: break-word;
}
```